test/extensions/INTEL/SPV_INTEL_function_pointers/CodeSectionINTEL/alias.ll

```diff
@@ -0,0 +1,49 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
Note: most of these tests are copies from SPV_INTEL_function_pointers/* tests with -spirv-emit-function-ptr-addr-space option added in reverse translation to check if addrspace(9) is generated. The only exception to this is function-pointer-dedicated-as.ll test